src/main/java/Alex.java

```diff
@@ -0,0 +1,281 @@
+                    Deadline d = (Deadline) task;
+                    sb.append("|").append(d.getDeadline());
+                } else if (task instanceof Event) {
+                    Event e = (Event) task;
+                    sb.append("|").append(e.getStart()).append("|").append(e.getEnd());
+                }
+                writer.write(sb.toString());
+                writer.newLine();
+            }
+        } catch (IOException e) {
+            System.out.println("Error saving tasks: " + e.getMessage());
+        }
+    }
+
+    /**
+     * Prints out the list of tasks.
+     */
+    private static void handleList() {
+        System.out.println("    ____________________________________________________________");
+        if (tasks.size() == 0) {
+            System.out.println("     You have nothing in your Todo. May...maybe we should hang out for a while...");
+        } else {
+            System.out.println("     Here are the tasks in your list:");
+            for (int i = 0; i < tasks.size(); i++) {
+                Task t = tasks.get(i);
+                System.out.println("     " + (i + 1) + ".[" + t.getTaskType() + "][" + t.getStatusIcon() + "] " + t);
+            }
+        }
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Marks a task (based on the number given after 'mark').
+     */
+    private static void handleMark(String input) {
+        try {
+            int index = Integer.parseInt(input.substring(4).trim());
+            if (index < 1 || index > tasks.size()) {
+                throw new IndexOutOfBoundsException();
+            }
+            tasks.get(index - 1).setMark();
+            System.out.println("    ____________________________________________________________");
+            System.out.println("     Sugoii~!! I've marked this task as done:");
+            System.out.println("       [" + tasks.get(index - 1).getTaskType() + "][" 
+                               + tasks.get(index - 1).getStatusIcon() + "] " 
+                               + tasks.get(index - 1).description);
+            System.out.println("    ____________________________________________________________");
+        } catch (Exception e) {
+            System.out.println("    BAKA! I bet you weren't listening to me again! Specify a valid task number to mark.");
+        }
+    }
+
+    /**
+     * Unmarks a task (based on the number given after 'unmark').
+     */
+    private static void handleUnmark(String input) {
+        try {
+            int index = Integer.parseInt(input.substring(6).trim());
+            if (index < 1 || index > tasks.size()) {
+                throw new IndexOutOfBoundsException();
+            }
+            tasks.get(index - 1).setUnmark();
+            System.out.println("    ____________________________________________________________");
+            System.out.println("     Gambatte, I've marked this task as not done yet:");
```

Comment:
Avoid usage of non-english words as this application caters to international audience